thirdparty/faiss/faiss/utils/distances.cpp

```diff
@@ -65,7 +65,9 @@ void fvec_norms_L2(
         size_t nx) {
 #pragma omp parallel for if (nx > 10000)
     for (int64_t i = 0; i < nx; i++) {
-        nr[i] = sqrtf(fvec_norm_L2sqr(x + i * d, d));
+        auto norm = fvec_norm_L2sqr(x + i * d, d);
```

Comment:
no, because it might use norm for distance = x^2 - 2xy + y^2 calculation, not for a cosine-related code